.github/workflows/detect-changes.yml

```diff
@@ -82,9 +77,16 @@ jobs:
       - name: Install package dependencies
         run: yarn install --frozen-lockfile --non-interactive
```

Comment:
this step and 'PR: Fetch base branch Git history' could that be moved into the ./.github/actions/affected-action?

Reply:
Other projects is using this `is-affected` action, and i have no idea what it does, so i would rather let the frontend guys refactor this when they think its needed